src/workerd/server/server.c++

```diff
@@ -1565,7 +1569,51 @@ private:
 
   kj::Promise<void> writeLogfwdr(uint channel,
       kj::FunctionParam<void(capnp::AnyPointer::Builder)> buildMessage) override {
-    KJ_FAIL_REQUIRE("no logging channels");
+    auto& context = IoContext::current();
+
+    auto headers = kj::HttpHeaders(context.getHeaderTable());
+    auto client = context.getHttpClient(channel, true, nullptr, "writeLogfwdr"_kjc);
+
+    kj::Url url;
+    url.scheme = kj::str("https");
+    url.host = kj::str("fake-host");
+    auto urlStr = url.toString(kj::Url::Context::HTTP_PROXY_REQUEST);
```

Comment:
Non-blocking nit: this feels a bit wasteful. I might be missing something but why not simply `auto urlStr = kj::str("https://fake-host");` here?